src/ParameterizedFunctionBase.cpp

```diff
@@ -190,14 +190,14 @@ void SetCoeffsInternal(unsigned int numCoeffs,
     if(coeffsDest.is_allocated()){
         if(coeffsSrc.size() != numCoeffs){
             std::stringstream msg;
-            msg << "Error in ParameterizedFunctionBase<MemorySpace>::SetCoeffs.  Expected coefficient vector with size " << numCoeffs << ", but new coefficients have size " << coeffs.size() << ".";
+            msg << "Error in ParameterizedFunctionBase<MemorySpace>::SetCoeffs.  Expected coefficient vector with size " << numCoeffs << ", but new coefficients have size " << numCoeffs << ".";
```

Comment:
This should stay as-is . `numCoeffs` is the number of coefficients in the PFB object, but `coeffs` are the new coefficients we want to include.